### PR TITLE
[refactor] Cape system rewrite 

### LINKED ISSUE #1102: Switch capes and rpc to 1 file
I'd rather have it all in 1 JSON for organizations sake. Also there's no reason for capes to be the full URL, just the 10 / 13 is enough.

Please do **not** work on this, I have to do it myself due to the spreadsheet I have to make, as the amount of capes / rpc users is inconsistent

### PR DESCRIPTION
**Pull request by l1ving**
**Describe the pull**
Rewrites the cape system and rendering to use [cape-api](https://github.com/kami-blue/cape-api).

Closes #1102

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/KamiMod.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
     public static final String DOWNLOADS_API = "https://kamiblue.org/api/v1/downloads.json";
     public static final String DONATORS_JSON = "https://raw.githubusercontent.com/kami-blue/assets/assets/assets/donators.json";
-    public static final String CAPES_JSON = "https://raw.githubusercontent.com/kami-blue/assets/assets/assets/capes.json";
+    public static final String CAPES_JSON = "https://raw.githubusercontent.com/kami-blue/cape-api/capes/test.json";
```
Comment (Xiaro) [inline src/main/java/me/zeroeightsix/kami/KamiMod.java:59]: Why is it still test.json :kannagun:

File: src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt [hunk 2]
```diff
@@ -22,100 +42,211 @@ import javax.net.ssl.HttpsURLConnection
...
+            partialTicks: Float
+    ): Boolean {
+        if (isDisabled
+                || entity !is AbstractClientPlayer
+                || entity.getItemStackFromSlot(EntityEquipmentSlot.CHEST).getItem() != Items.ELYTRA) return false
+
+        val cape = capeUsers[entity.gameProfile.id]
+
+        return if (cape != null) {
+            renderElytra(renderer, model, entity, cape, limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scale, partialTicks)
```
Comment (Xiaro) [inline src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt:176]: I think we should inline it for less ♿ parameter spam

File: src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt [hunk 3]
```diff
@@ -22,100 +42,211 @@ import javax.net.ssl.HttpsURLConnection
...
+        GlStateManager.color(color.r / 255.0f, color.g / 255.0f, color.b / 255.0f, 1.0f)
+        renderer.bindTexture(texture.location)
+        GlStateManager.pushMatrix()
+        GlStateManager.translate(0.0f, 0.0f, 0.125f)
+        model.setRotationAngles(limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scale, player)
+        model.render(player, limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scale)
+
+        if (player.getItemStackFromSlot(EntityEquipmentSlot.CHEST).isItemEnchanted) {
+            LayerArmorBase.renderEnchantedGlint(renderer, player, model, limbSwing, limbSwingAmount, partialTicks, ageInTicks, netHeadYaw, headPitch, scale)
+        }
```
Comment (Xiaro) [inline src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt:235]: Potential enchant glint bug here. Need some test I guess. If it breaks maybe only render enchant glint for the last layer?

**Review by Xiaro — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/KamiMod.java (hunk 1 above)
Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/KamiMod.java:59]: I know, I'll fix

File: src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt (hunk 2 above)
Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/module/modules/client/Capes.kt:176]: mfw

**Review by Xiaro — APPROVED** [review]

**Pull request merged**